zavit.Web.Mvc/App/Modules/places/placesClient.js

```diff
@@ -0,0 +1,15 @@
+﻿import * as ApiSettings from "../settings/apiSettings";
+
+export function getPlaces(latitude, longitude, radius, name) {
+    console.log(name);
+    const placesUrl = `${ApiSettings.apiUrl}/api/places?latitude=${latitude}&longitude=${longitude}&radius=${radius}&name=${name}`;
```

Comment:
`${ApiSettings.apiUrl}api

No forwardslash needed, I think I left it somewhere as well and it ends up being double forwardslash, if you know where else it is please remove